src/frontend/apps/desk/src/pages/teams/create.tsx

```diff
@@ -55,11 +57,7 @@ const Page: NextPageWithLayout = () => {
             onChange={(e) => setTeamName(e.target.value)}
             rightIcon={<span className="material-icons">edit</span>}
           />
-          {isError && (
-            <Text className="mt-s" $theme="danger" $textAlign="center">
-              {t('Something bad happens, please retry.')}
-            </Text>
-          )}
+          {isError && error && <TextErrors causes={error.causes} />}
```

Comment:
Should we use a form library which would handle for user form submission error ?

Reply:
Let's open an issue, the error returned by the back is not explicit for the user.
<img width="453" alt="Capture d’écran 2024-02-19 à 19 02 22" src="https://github.com/numerique-gouv/people/assets/45729124/83926992-026c-42c9-8ce9-3976820ccc82">

Reply:
If you have a precise idea, please open an issue. As the slug is an internal mechanism (the user never types it), the error message should be ""A team with this name already exists."